nbs/15_migrate.ipynb

```diff
@@ -28,7 +28,8 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                ss = listify(cell['source'])
</pre><p>Why are these needed? Shouldn't it be handled by NbCell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Thanks yes this a bit obtuse, not necessary, thanks for flagging this.